CHANGELOG.md

```diff
@@ -1,3 +1,46 @@
+### citus v10.2.0 (September 11, 2021) ###
+
+* Fixes a bug that may cause crash while aborting transaction
+
+* Fixes bug about int and smallint sequences on MX
+
+* Fixes a bug that caused worker_append_table_to_shard to write as superuser
+
+* Enable transferring of data using binary encoding by default on PG14
+
+* Add helper UDFs for easy time partition management
```

Comment:
should we mention UDF names here, I think so

Reply:
Added those in a multiline entry